### ISSUE TITLE
Unable to enable plugin 4.0.0-alpha in godot4 beta 8

### ISSUE THREAD

**Issue by svn133** [MikeSchulze/gdUnit4#49]
**The used Godot version:**
v4.0.beta8.official

**OS including version:**
macOS 12.5.1

**Describe the bug**
Unable to enable gdUnit4 4.0.0-alpha, after downloading it via AssetLib. When trying to enable it, an error is displayed instead.
See screenshot with error, also the errors in the Output log:
```
Transient parent has another exclusive child.
core/config/project_settings.cpp:348 - Property not found: editor_plugins/enabled
```
![gdunit4-enable-error](https://user-images.githubusercontent.com/114500633/206909133-ecfda023-1fc5-4d6e-8b05-c2408d4c62a0.png)

Not sure if this is already fixed in the upcoming release, so filing this bug here just in case.

**Steps to Reproduce**
1. open any project, or create a new one
2. goto AssetLib
3. download gdUnit4 4.0.0-alpha
4. wait until it succeeds
5. open project settings and try to enable the plugin

**Minimal reproduction project:**
just create a new project

**Comment by MikeSchulze** [comment]
allready fixed by https://github.com/MikeSchulze/gdUnit4/commit/ff82bb865a4d9eca51cb677da1e90abeca2217d9
Will be released with GdUnit 4.0.1-beta in the next minutes

**Comment by git2013vb** [comment]
@MikeSchulze The same error happen to me 
I have v4.0.beta12.official [3c9bf4bc2]
Debian 11

Thanks  :) It is an awesome plugin :)) I can't wait to be able to use it
